scripts/firedrake-install

```diff
@@ -1615,20 +1615,21 @@ if mode == "install":
             log.info("Pip installing %s to venv" % package)
             run_pip_install(package.split())
 
-    # Temporary workaround for missing VTK wheel on Python 3.9
-    if sys.version_info[:2] == (3, 9):
-        source = "https://github.com/firedrakeproject/VTKPythonPackage/releases/download/firedrake_20210113/"
+    # "Temporary" workaround for missing VTK wheel on Python 3.10
+    if sys.version_info[:2] == (3, 10):
+        source = "https://github.com/firedrakeproject/VTKPythonPackage/releases/download/firedrake_20220106/"
```

Comment:
Should be https://github.com/firedrakeproject/VTKPythonPackage/releases/tag/firedrake_20220106?